cli/tests/integrations/test_config.py

```diff
@@ -295,6 +314,7 @@ def test_timeout(env):
 def test_parse_error():
     env = os.environ.copy()
     path = os.path.join('tests', 'data', 'config', 'parse_error.toml')
+    os.chmod(path, 0o600)
```

Comment:
Git doesn't store file permissions (only the executable bit) and I wanted to minimize configuration for running the tests locally.